metricflow-semantics/metricflow_semantics/model/semantics/element_filter.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import FrozenSet
+
+from typing_extensions import Self, override
+
+from metricflow_semantics.collection_helpers.merger import Mergeable
+from metricflow_semantics.model.linkable_element_property import LinkableElementProperty
+
+
+@dataclass(frozen=True)
+class LinkableElementFilter(Mergeable):
```

Comment:
Should we call this `LinkableElementPropertyFilter`? On first glance I would have assumed this is referring to a filter on a group by spec or something similar to that.

Reply:
This actually gets updated in a later PR to filter by element name, so I kept this more general.